.github/workflows/elixir-build-check.yaml

```diff
@@ -114,3 +114,8 @@ jobs:
           git diff --name-only "$default_branch_ref" | \
           egrep '(\.ex|\.exs)+$' | \
           xargs --no-run-if-empty mix format --check-formatted --dry-run
+  single-commit-check:
```

Comment:
I'm not sure this really belongs in the `elixir-build-checks` workflow. It's not Elixir specific.

Reply:
See the discussion in #elixir-dev in slack